event-adapter/README.md

```diff
@@ -0,0 +1,181 @@
+# 1. Create a cluster ensuring that Stackdriver Logging API are enabled (both reading and writing)
```

Comment:
"is enabled (both for reading and writing)"

Also, please Include brief overview and create a section to indicate that this is a guide how to run Event Adapter on GCP cluster.